docs/devguide.md

```diff
@@ -25,7 +25,12 @@ layout:
     │   └── apiserver           # The service catalog API server binary
     │   └── controller-manager  # The service catalog controller manager binary
     ├── contrib                 # Contains all non-essential source
```

Comment:
What do you think if we just spell out what is in there. For example, "build scripts, sample or mock code for broker..."